test/test_quantized.py

```diff
@@ -24,13 +24,58 @@ def _dequantize(qx, scale, zero_point):
     return x
 
 
-@skipIfNotRegistered("Relu_ENGINE_DNNLOWP",
+# Make sure we won't have overflows from vpmaddubsw instruction used in FBGEMM
```

Comment:
Please add some comments explaining what is going on inside this function. It looks like this function is modifying the weights to eliminate overflow, can you elaborate a bit?

Reply:
Will elaborate the function.